contributors/guide/scalability-good-practices.md

```diff
@@ -1,7 +1,9 @@
 ---
 title: "Scalability Good Practices"
 weight: 13
-slug: "scalability-good-practices" 
+description: |
+  Some programming best practices and examples to help you avoid potential
+  scalability regressions.
```

Comment:
(In a different PR) Would it make sense to move this to a sig-scalability under devel? These are very specific best practices and don't really need to be in the contributor guide.

Reply:
Yeah, I was thinking about that one too when adding it =/
I'll go ahead and remove the header then move it in another PR.